README.md

```diff
@@ -10,3 +10,67 @@ The Mattermost operator is in alpha, data loss might occur.
 Mattermost is a scalale, open source collaboration tool. It's written in Golang and React.
 
 The idea of this project is to build the Kubernetes Operator for Mattermost to mak more easy for Kubernetes users deploy and manage your Mattermost instance.
+
+
+## Getting Started
+
+### Prerequisites
+
+- Kubernetes cluster 1.8.0+.
+- `kubectl` configured to refer to relevant Kubernetes cluster.
+
+### Create Operator and related resources
+
+To start Mattermost-Operator, we need to install the dependencies first.
+
+#### MySQL-Operator
+To install MySQL-Operator apply the manifests that you can find in the `docs` folder
+
+```
+$ kubectl create ns mysql-operator
+$ kubectl apply -n mysql-operator -f https://github.com/mattertmost/mattermost-operator/blob/master/docs/mysql-operator/mysql-operator.yaml?raw=true
```

Comment:
This URL doesn't resolve for me -- first, I note a typo `mattertmost`, but even then I can't find the file.

Reply:
Yeah, I think those files are currently located in https://github.com/mattermost/mattermost-operator/tree/master/test/mysql

From my work making the provisioning server deploy the mattermost operator, I have a todo item that includes getting the K8s yaml resources in one spot, so maybe we should do that here with the documentation?